pkg/resources/sequence.go

```diff
@@ -40,11 +41,13 @@ var sequenceSchema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Required:    true,
 		Description: "The database in which to create the sequence. Don't use the | character.",
+    ForceNew:    true,
```

Comment:
these force news will cause the sequence count(next val) to reset when these are updated. is that expected/wanted?

Reply:
yes for a new database or schema it follows the same path that other tf-snowflake resources follow

Reply:
actually it won't reset it, but it will rebuild it with the original value
```
nextValue, err := strconv.Atoi(sequence.NextValue.String)
	if err != nil {
		return err
	}

sq.WithStart(nextValue)
```